### PR TITLE
added new feature for model evaluation

### PR DESCRIPTION
**Pull request by qingpeng**
#### using scikit-learn for metric calculation,  it is possible to use built-in metric API in tensorflow to do this, but may not be suitable for this case.

https://www.tensorflow.org/api_docs/python/tf/metrics

https://www.tensorflow.org/extend/estimators

#### using a simple testing set, the evaluation result is 

> more evaluation_result.txt
accuracy:0.942025703448
Confusion_matrix:
4630,0,0,0
0,3148,486,511
0,5,4843,0
0,22,0,4018
virus_f1_score:0.860109289617

### CODE REVIEW

**Comment by arivers** [comment]
Could you add sklearn to the setup.py and requirements.txt so travis-ci can make sure the build passes?
